examples/bri-3/components/identity/bpiAccount/bpiAccount.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface IBpiAccount {
+    id: string; // TODO: Add uuid after #491
+    nonce: number;
+    ownerBpiSubjectId: string;
```

Comment:
that should be an array in order to allow for flexibility and extensibility. Plus typically, a BPI account will have two owners since it connects to a joint state.

Reply:
Could you please elaborate on the second sentence. Why should it have two owners typically? Isn't it more common that org A would work with one BpiAccount and org B with another?

Reply:
I agree that it should be an array, multiple owners makes sense in multisig scenarios.

Reply:
Perhaps we should we should discuss and conclude a scope of functionality for alignment of team and maintainers in future PRs. Originally the team agreed on the "simplest implementation" to cover our intended use case. We seem to ever so slowly be drifting towards accommodating common use case needs that fall outside the "MUST" requirements of the standard; a direction in which has my support.

Reply:
If we want the SRI to be useful for multi party coordination WITHOUT being clunky aka you need to double or triple the BPI accounts. And will impact adoption. 

I agree we should keep it as simple as possible while ensure good usability.